scripts/py_matter_idl/matter_idl/generators/kotlin/MatterClusters.jinja

```diff
@@ -210,7 +210,7 @@ class {{cluster.name}}Cluster(private val controller: MatterController, private
 {% if command.is_timed_invoke -%}
```

Comment:
could we indent if/else blocks above given that the usage of `-` will collapse the output text anyway? Without indentation, it seems really hard to figure out where the if/else comes from. In this particular case I was wondering why the timedInvoke logic was duplicated and it was hard to figure out (seems to be based on command.input_param ... but then why don't we just drop the `{%- if not loop.last -%}, {% endif %} ` so we always end in a `,` and then timed invoke logic is shorter.

Reply:
Thinking:

```jinja
{%- if command.input_param -%}
   {%- for field in (cluster.structs | named(command.input_param)).fields -%}
      {{field.name | lowfirst_except_acronym}}: {{encode_value(cluster, field | asEncodable(typeLookup), 0)}},
   {%- endfor -%}
{%- endif -%}
  timedInvokeTimeoutMs: Duration {%- if !command.is_timed_invoke -%}? = null {%- endif -%}
)
```

which seems a lot shorter

Reply:
Acked